npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -33,7 +33,10 @@ export async function makeWebpackConfig (userWebpackConfig: webpack.Configuratio
 
   const entry = path.resolve(__dirname, './browser.js')
 
+  // The second line here replaces backslashes on windows with posix compatible slash
+  // See https://github.com/cypress-io/cypress/issues/16097
   const publicPath = path.join(devServerPublicPathRoute, '/')
+  .replace(/\\/g, '/')
```

Comment:
Replacing like this is not safe. Backslashes are valid in POSIX paths. Instead, you should do this
`string.split(path.sep).join('/')` or `string.replace(path.sep, '/')`

Reply:
OK, I will update.